docker/README.md

```diff
@@ -1,180 +1,5 @@
-# Open AD Kit: Containerized Workloads for Autoware
+## Open AD Kit: Containerized Workloads for Autoware
```

Comment:
```suggestion
# Open AD Kit: containerized workloads for Autoware
```

Top level should always be single `#` and in overall repository we use [Sentence Case](https://www.scribbr.com/academic-writing/sentence-case/) for all titles.